src/pages/index.js

```diff
@@ -1,8 +1,74 @@
 import React from 'react'
 import { Helmet } from 'react-helmet'
 import s from 'styled-components'
+//import MDEditor from '@uiw/react-md-editor';
+import InputField from '../components/InputField.js'
+import { Navbar, Button } from 'react-bootstrap'
 
 const IndexPage = () => {
+  const fullTemplate = require('../templates/json/34st-2021s.json')
```

Comment:
`require` is how we import things in Express backend, in frontend React, convention is to use `import`. Check this out for how to use `import` to import JSON in GatsbyJS. Again, since this is an internal import, it sld be placed outside of the component and right below the internal imports code block